### DISCUSSION TITLE
A what?

### DISCUSSION THREAD

**Discussion by ShiroWasDreaming** [PositionExchange/position-protocol#161] [General]
https://github.com/PositionExchange/position-protocol/blob/4864959eec03bc98312ae34797a53bc7ec3f3e60/scripts/helper.ts#L41
child process detecting, xd

**Comment by JustInPosition** [comment]
Only execute the `node_modules` binary. Not using yet btw xD

**Reply by ShiroWasDreaming** [reply]
:>

**Comment by RavEthaners** [comment]
https://github.com/PositionExchange/position-protocol/blob/4864959eec03bc98312ae34797a53bc7ec3f3e60/hardhat.config.ts#L106
is that generic api key?

**Reply by ShiroWasDreaming** [reply]
gen-api
